### PR TITLE
Local-storage add-on : Reset all sections button functionality

### LINKED ISSUE #39: "Clear All" kind of functionality to clean up the added readme sections to start fresh
Currently , when a user wants to reset its readme file's content (i.e start from scratch with just the 'title and description' template) , he/she has to refresh the entire page in chrome/any browser . Currently we can delete individual sections by clicking on delete icon one by one , but can't delete all at one . So , should we support a "clear all" to clean up all sections at once or let us just browser refresh do it for now ?  Also , is the browser refresh supposed to clean-up all the used sections while a user is creating a readme , and in middle of it the progress is lost if he just refreshes the browser (if he didn't backed up the document by downloading in-between) ?  While working with swagger editor online for designing APIs i have experienced , it saves the state for the current session of user from local storage with any custom changes , even if he refreshes browser as here : https://editor.swagger.io/ .

image

### PR DESCRIPTION
**Pull request by saurabhchaturvedy**
resolves #39 

Hi @katherinepeterson , as we had discussed in issue #39 , that we will implement a clear all/reset button to clear all sections to start afresh after local storage is implemented .  Now that i implemented local-storage in PR #80 , I have implemented reset functionality as below and its live on preview url  for you to check . 

image

I have also added translations for the word 'Reset' per language (best of my efforts via Google Translate :) , if this require any minor change someone can work on this via a separate issue . ) 

Request you to consider to merge this PR first , as its a feature enhancement & will avoid any minor conflicts with any translations PR .
Once you approve this PR , i will look at the issue #81 .

Thanks !!

### CODE REVIEW

**Comment by saurabhchaturvedy** [comment]
@katherinepeterson any comments on this MR, or do i need to change something ? Let me know, as i see conflict in one file on this MR  , but resolve conflicts button is disabled for me . Thanks

**Comment by rohitjmathew** [comment]
@saurabhchaturvedy can we reset it back to initial state of only Title and Description slug?

File: components/SectionsColumn.js [hunk 1]
```diff
@@ -108,6 +127,16 @@ export const SectionsColumn = ({
     <div className="sections w-80">
       <h3 className="px-1 text-sm font-medium border-b-2 border-transparent text-emerald-500 whitespace-nowrap focus:outline-none">
         {t('section-column-section')}
+        <span className="pl-2 float-right">{t('section-column-click-reset')}</span>
```
Comment (katherinepeterson) [inline components/SectionsColumn.js:130]: Can we move the "Reset" span inside the button so it is also clickable?

**Review by katherinepeterson — CHANGES_REQUESTED** [review]
Sorry for the delay, and thanks for taking this on, it is a highly requested feature.

A few things:
1. I left a comment about making the word "Reset" clickable in addition to the logo. 
2. I agree with @rohitjmathew's comment about Resetting to the initial state as the first time user's experience, which is having the only the "Title and Description" section instead of none
3. Can you make sure all the languages are covered for the reset button translation? Not sure but there may have been more languages added
4. If you could resolve conflicts that would be great, if the button is disabled it just means you have to do it locally instead of from the GitHub UI. If you can't, I can take care of it.

Thank you, great work!

File: components/SectionsColumn.js (hunk 1 above)
Comment (saurabhchaturvedy) [reply components/SectionsColumn.js:130]: @katherinepeterson  yes , will do this .

**Comment by saurabhchaturvedy** [comment]
@katherinepeterson thanks for the review , i can resolve the conflicts , just wanted to know if the comments that you have given , should i resolve them in this same PR or work of them as separate issue, as this MR was pending for some time  ?

File: components/SectionsColumn.js (hunk 1 above)
Comment (saurabhchaturvedy) [reply components/SectionsColumn.js]: this is fixed @katherinepeterson

**Pull request closed without merging**